Braintree Component/BraintreeComponents/Scripts/BraintreeHostedFields.ts

```diff
@@ -28,6 +28,8 @@ namespace $DP.Control {
         hostedFieldsCreatedInstance: any;
 
         // Returns the Control HTML element
+        
+        // @ts-ignore
```

Comment:
You just added ts-ignore here? was it still pulling in the wrong version of jquery types, maybe?

Reply:
I think we need to do a couple of things here:
1. Probably the version of _jQuery_ is checked in with all the typings that are checked in here, and we probably need to replace checked in typings with those from a more recent version of decisions
2. Add a package.json to this thing that specifies the version of TypeScript contemporary to the version of Decisions it will be used in
    - we shouldn't be relying on the right one being installed ambiently by chance
3. add a build script to that package.json, and have the .csproj file call that, so the local version of TypeScript is used

Reply:
For the ts-ignore, I'm not a fan of that either. It's definitely not the best solution. But it was becoming a time waster and I needed the build script to run. Now that the Braintree module loads, I can circle back and resolve the jQuery type conflict.  

As per the "couple of things"

1. We can definitely try this. From what I could tell, it seemed like the issue was the jQuery was defined as a type in many locations. I think this file didn't know which type reference to use
2. I did this and forgot to copy it into the repo. I'll put that back in.
3. Sure. I can do that

Reply:
hopefully deleting checked-in jquery types, and installing the correct jQuery types in the package.json will solve this problem